### PR TITLE
daemon: Handle SIGTERM and SIGKILL

### PR DESCRIPTION
**Pull request by mrogers950**
Just pushing what I came up with at the moment. Still need to test.

### CODE REVIEW

File: cmd/manager/daemon.go [hunk 1]
```diff
@@ -264,6 +266,8 @@ func daemonMainLoop(cmd *cobra.Command, args []string) {
 
 	errChan := make(chan error)
 	ctx, cancel := context.WithCancel(context.Background())
+	sigTermChan := make(chan os.Signal)
+	signal.Notify(sigTermChan, syscall.SIGTERM, syscall.SIGKILL)
```
Comment (JAORMX) [inline cmd/manager/daemon.go:270]: Would [NotifyContext](https://golang.org/pkg/os/signal/#NotifyContext) be useful here?

Comment (mrogers950) [reply cmd/manager/daemon.go:270]: It would indeed! I need to update deps to 1.16 first

**Pull request merged**